docs/dnsupdate.rst

```diff
@@ -1,519 +0,0 @@
-Dynamic DNS Update (RFC2136)
```

Comment:
I presume removing this file was a mistake :)

Reply:
Indeed! I restored the file.